topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml

```diff
@@ -0,0 +1,596 @@
+      second the group (sample) name, in our case "pampa" or "anguil".
+    position: left
+  - title: Optimization of files for computation
+    content: >-
+      Because we are sequencing many of the same organisms, we anticipate that
+      many of our sequences are duplicates of each other. Because it's
+      computationally wasteful to align the same thing a bazillion times, we'll
+      unique our sequences using the "Unique.seqs" command
+    backdrop: true
+  - title: Remove duplicate sequences
+    element: '#tool-search-query'
+    content: Search for the "Unique.seq" tool and click on the "Unique.seqs" tool
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_unique_seqs%2Fmothur_unique_seqs%2F1.36.1.0"]
+  - title: Remove duplicate sequences
+    element: '#center-panel'
+    content: |-
+      Execute the tool with <ul>
+        <li>"fasta" to the merged fasta file</li>
+      </ul> 
+    position: left
+  - title: Remove duplicate sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      This "Unique.seqs" tool outputs two files, one is a fasta file containing
+      only the unique sequences, and a names files. The names file consists of
+      two columns, the first contains the sequence names for each of the unique
+      sequences, and the second column contains all other sequence names that
+      are identical to the representative sequence in the first column.<br><br>
+      How many sequences were unique? How many duplicates were removed?
+    position: left
+  - title: Count sequences
+    element: '#tool-search-query'
+    content: Search for the "count.seqs" tool and click on the "Count.seqs" tool
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_count_seqs%2Fmothur_count_seqs%2F1.36.1.0"]
+  - title: Count sequences
+    element: '#center-panel'
+    content: |-
+      Execute the tool with <ul>
+        <li>"name" to the name file from "Unique.seqs"</li>
+        <li>"Use a group file" to "yes"</li>
+        <li>"group" to the group file from "Make.group"</li>
+        <li>selection of the two groups</li>
+      </ul> 
+    position: left
+  - title: Count sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      The "Count.seqs" file keeps track of the number of sequences represented
+      by each unique representative across multiple samples. We will pass this
+      file to many of the following tools to be used or updated as needed.
+    position: left
+  - title: Quality Control
+    content: >-
+      The first step in any analysis should be to check and improve the quality
+      of our data.<br><br> For more information on the topic of quality control,
+      please see our training materials <a
+      href="https://galaxyproject.github.io/training-material/NGS-QC/">here</a>.
```

Comment:
link broken, replace with either
https://galaxyproject.github.io/training-material//topics/sequence-analysis/tutorials/quality-control/tutorial.html for the tutorial, or link to the topic https://galaxyproject.github.io/training-material//topics/sequence-analysis for easy access to both slides and tutorial